data/hai/culture hai.txt

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+############################################
+#          No Translator Required          #
+############################################
+
+mission "ZZCU Theater Chat"
```

Comment:
It seems to me like the strict offer conditions of most of these missions fulfil the purpose of making sure they don't get offered over other missions. Even ignoring that though, I don't see an issue with sometimes getting these missions over other missions. It's not like these ones repeat. Therefore, I wouldn't have these extra characters in the mission names just to push the mission to the end of the offer list. 

I'd remove the comments, like the one above. They seem unnecessary to me. There's no need to comment to organize these missions, or to say who worked on them, since we know that from the PR and ultimately the changelog attribution. Whatever the case on comments, we should have ones like the "No Translator Required" comment above, as that actually goes against the coding style guide, so I'd avoid it in the data files as well.

> Fancy, decorated comments should not be used.
> For example, perhaps your editor has a "feature" that lets you automatically insert comments like this:
```
/*************************************
 * My fancy comment.
 * This is a comment that could have
 * been a single line without these
 * fancy decorative elements.
 *************************************/
```
> But, those two lines of asterisks are eye-grabbing and completely unnecessary on any editor that supports syntax coloring, and more importantly anyone using a different editor would have to manually type an asterisk at the beginning of every comment line and figure out exactly how many asterisks are in those rows at the top and bottom, in order to be consistent with your code.

Having separate files just for these missions also seem unnecessary.  These could simply be put into the existing mission files. "culture hai.txt" gets added to "hai mission.txt," the rest just get added to "transport mission.txt" (even though they aren't actually transport missions, but that file has already had missions added that aren't transport related).

Reply:
"transport missions.txt" = "general purpose mission file for non specific missions"

Reply:
The ZZ was mostly because the Spider had raised some strident objections to them being in a position to block other missions, since these are all missions that are specifically of no consequence.

This was also why they're in their own files. There's going to be more of them, and all the other missions in the game so far have some sort of consequence. Whether it's a reward, or a change in the world, or they unlock other missions or dialogue options. All the other missions _do_ something, and these only provide world-building insight to the player. There's only 23 missions across the set so far, but Lake is doing some more and the plan is to come back with a second batch and so on until (across this scope) there's about 180 of them. 

I really think it's better if we don't put a clutter of no-consequence fluff missions in the same files with the ones that actually do things. If only for the benefit of content creators and their sanity when trying to sort through things.

Reply:
If we want these is separate files, then at the very least combine all the human ones. The distinction between regions/sources into different files is overkill to me. And how about instead of calling them “culture missions,” we call them “spaceport conversations.txt,” with the Hai one being “hai spaceport conversations.txt”?